test/corpus/create.txt

```diff
@@ -805,80 +706,17 @@ CREATE UNLOGGED TABLE tableName (
         (column_definition
           (identifier)
           (numeric
-            (keyword_numeric)))))))
-
-================================================================================
-Create table with index
-================================================================================
```

Comment:
Looks like this test was removed by mistake.

Reply:
Yes, I fixed that. Sorry.